src/js/main.js

```diff
@@ -1,6 +1,62 @@
-import {animatedScrollTo} from './es6-scroll-to.js';
+// const buttons = document.querySelectorAll('.button');
+// buttons.forEach(btnClick => btnClick.addEventListener('click', openInfobox));
 
-document.querySelector('.title').innerHTML = 'Great success!'
-document.querySelector('.text').innerHTML = 'Javascript is working!'
 
-animatedScrollTo(500);
\ No newline at end of file
+// function openInfobox(event) {
+//     const btn = event.target;
+//     const elementBlockSelector = 'contactUs';
+//     const open = '--opened';
+//     const allContactUs = document.querySelectorAll('.contactUs');
+//     const ctaSection = '.ctaSection';
+
+
+//     for (let i = 0; i < allContactUs.length; i++) {
+//         const element = allContactUs[i];
+
+//         if (btn.closest(ctaSection).querySelector('.contactUs') !== element) {
+//             element.classList.remove(elementBlockSelector + open);
+//         }
+//     };
+
+//     btn.closest(ctaSection).querySelector('.contactUs').classList.toggle(elementBlockSelector + open);
+// };
+
+
+
+// ---ES6 way of doing the loop---
+// console.log(allContactUs.forEach);
+// allContactUs.forEach(function(element){
+//     console.log(element);
+//     element.classList.remove(elementBlockSelector + open);
+// });
+
+
+
+var buttons = document.getElementsByClassName('button');
+
+for (var i = 0; i < buttons.length; i++) {
+    const element = buttons[i];
+
+    element.addEventListener('click', openInfobox)
+};
+
+function openInfobox(event) {
+    var btn = event.target;
+    var elementBlockSelector = 'contactUs';
+    var open = '--opened';
```

Comment:
Consistent naming of variables. This variable holds a modifier, so the variable name should indicate that. I suggest `openModifier`.